kotlin/display-device-location/src/main/java/com/esri/arcgisruntime/sample/displaydevicelocation/MainActivity.kt

```diff
@@ -97,7 +99,7 @@ class MainActivity : AppCompatActivity() {
   }
 
   /**
-   * Request fine and coarse location permissions for API level 23+.
+   * Request fine and coarse location permissions for API level 23+
```

Comment:
we should leave the periods in here, since for class or method doc, it should be a full sentence. It's only for in-method code that we shouldn't use periods.

Reply:
same for where the period is removed elsewhere

Reply:
My bad haha, I'll change those back.